dhis-2/dhis-services/dhis-service-core/src/main/resources/org/hisp/dhis/eventchangelog.hiberante/EventChangeLog.hbm.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<!DOCTYPE hibernate-mapping PUBLIC
+  "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+  "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+
+<hibernate-mapping>
+  <class name="org.hisp.dhis.tracker.export.event.EventChangeLog" table="eventchangelog">
+
+    <id name="id" column="eventchangelogid">
+      <generator class="sequence">
+        <param name="sequence_name">eventchangelog_sequence</param>
+      </generator>
+    </id>
+
+    <many-to-one name="event" class="org.hisp.dhis.program.Event"
+      column="eventid" foreign-key="fk_eventchangelog_eventid" not-null="true"/>
+
+    <many-to-one name="dataElement" class="org.hisp.dhis.dataelement.DataElement" column="dataelementid"
+      foreign-key="fk_eventchangelog_dataelementid" />
+
+    <property name="eventProperty" length="100" />
+
+    <property name="currentValue" length="50000" />
+
+    <property name="previousValue" length="50000" />
+
+    <property name="changeLogType" column="changelogtype" length="100" not-null="true">
+      <type name="org.hibernate.type.EnumType">
+        <param name="enumClass">org.hisp.dhis.changelog.ChangeLogType</param>
+        <param name="useNamed">true</param>
+        <param name="type">12</param>
+      </type>
+    </property>
+
+    <property name="created" type="timestamp" not-null="true"/>
+
+    <property name="createdByUsername" column="createdby" type="string"/>
```

Comment:
Is this used?

Reply:
yes, it is used when creating a new entry and when fetching the results it's used as a condition for the join to the user table